certified-connectors/ZellisIntelligencePlatform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1063 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Zellis Intelligence Platform",
+    "description": "This connectors consists various actions you can use to perform your daily tasks.",
+    "contact": {
+      "name": "Zellis Support",
+      "url": "https://zellis.service-now.com/csm",
+      "email": "ukmmprodmgmt@zellis.onmicrosoft.com"
+    },
+    "version": "1.0"
+  },
+  "host": "resourcelinkdev.co.uk",
```

Comment:
@basha-k We do not certify a connector which host URL is a staging or test one. Please let us know if the provided host URL is a production one.

Reply:
@basha-k We do not certify a connector which host URL is a staging or test one. Please let us know if the provided host URL is a production one.